### PR TITLE
Day 11

### PR DESCRIPTION
**Pull request by anubhav201241**
Added **Day 11** Files of 30 Days of ML

Topic: Machine Learning Competition: **Housing Price**  in **Machine Learning**

### CODE REVIEW

File: Day 11/README.md [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<h1>Day 11</h1>
+Topic:  Machine Learning competition: Housing Price  in <b>Machine Learning</b> 
+
+<h4>Kaggle: https://www.kaggle.com/anubhav202/housing-prices-competition</h4>
```
Comment (amino19) [inline Day 11/README.md:4]: @anubhav201241 : Can you pl change this link, to the challenge description link?

& also, at your previous PR! :slightly_smiling_face:

**Review by amino19 — CHANGES_REQUESTED** [review]

File: Day 11/README.md (hunk 1 above)
Comment (anubhav201241) [reply Day 11/README.md:4]: Sure :)

File: Day 11/README.md [hunk 2]
```diff
@@ -0,0 +1,4 @@
+<h1>Day 11</h1>
+Topic:  Machine Learning competition: Housing Price  in <b>Machine Learning</b> 
+
+
```
Comment (amino19) [inline Day 11/README.md:4]: @anubhav201241 : Do you mind adding the link as listed <a href= "https://github.com/DevsTrons/30-Days-of-ML/blob/master/Day%209/README.md">here</a>? 

Also same to <a href="https://github.com/DevsTrons/30-Days-of-ML/blob/master/Day%2010/README.md">Day 10</a> as well! :eyes:

**Review by amino19 — CHANGES_REQUESTED** [review]

File: Day 11/README.md (hunk 2 above)
Comment (anubhav201241) [reply Day 11/README.md:8]: Yes, I have made the changes :))

**Comment by amino19** [comment]
@anubhav201241 : Thanks for contributing! :partying_face:

**Comment by anubhav201241** [comment]
Welcome 😊🙌

**Pull request merged**